doc/tutorials/gpu-passthru.rst

```diff
@@ -0,0 +1,147 @@
+.. _gpu-passthrough:
+
+Enable GVT-d in ACRN
+#########################################
+
+This tutorial describes how to enable GVT-d in ACRN.
+
+Introduction
+*****************
+Intel GVT-d is one flavor of graphics virtualization approaches also
+names as Intel-Graphics-Device pass-through feature.
+It based on Intel VT-d technology,
+and some special graphics related configuration.
+This flavor allows direct assignment of an entire GPU’s prowess to a single user,
+passing the native driver capabilities through the hypervisor without any limitations.
+
+Hardware Platform Supported
+***************************
+Currently, ACRN has enabled GVT-d on the following platforms:
+
+* Kaby Lake
+* Whiskey Lake
+* Elkhart Lake
+
+BIOS Setting
+**********************
+
+Kaby Lake Platform
+==================
+* set "IGD Minimum Memory" to "64MB" in "Devices → Video → IGD Minimum Memory".
+
+Whiskey Lake Platform
+=====================
+* set "PM Support"  to "Enabled" in "chipset → System Agent (SA) Configuration → Graphics Configuration → PM support".
+* set "DVMT Pre-Allocated" to "64MB" in "chipset → System Agent (SA) Configuration → Graphics Configuration → DVMT Pre-Allocated"
+
+Elkhart Lake Platform
```

Comment:
Do we have any public info/link to that board? (It's not on the "Supported HW" page in the ACRN documentation and may be good to add if we can).

Reply:
"Supported HW" page in the ACRN documentation will add Elkhart Lake in the future. So we don't need to add  info/link to Elkhart Lake.

Reply:
@wenlingz @NanlinXie  Hi, nanlin and wenlin. 
`"Supported HW" page in the ACRN documentation will add Elkhart Lake in the future. `
Is it right?

Reply:
Agree, we'll add more info later, not blocking this PR.